annotated.go

```diff
@@ -285,10 +294,19 @@ var _ Annotation = resultTagsAnnotation{}
 //
 // If there has already been a ResultTag that was applied, this
 // will return an error.
+//
+// If the tag is invalid and has mismatched quotation for example,
+// (`tag_name:"tag_value') , this will return an error
 func (rt resultTagsAnnotation) apply(ann *annotated) error {
 	if len(ann.ResultTags) > 0 {
 		return errors.New("cannot apply more than one line of ResultTags")
 	}
+	for _, tag := range rt.tags {
+		if strings.Contains(tag, "'") {
+			correctedTag := strings.ReplaceAll(tag, "'", "\"")
+			return errors.New("the ResultTags tag has invalid quotations, did you mean " + correctedTag + " ?")
```

Comment:
Do we want to surround `correctedTag` with quotes or backticks to help demarcate it in the error message?

Reply:
Oh yeah, good catch thanks!